content/blog/2023-12-11-BrilCpp.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "C++ Infrastructure for Bril"
+[[extra.authors]]
+name = "Albert Xiao"
+[[extra.authors]]
+name = "Ryan Mao"
+[extra]
+bio="""
+Albert Xiao is an M.Eng student studying CS. He is particularly interested in systems and compilers development. 
+Ryan Mao is an M.Eng student studying CS. He is also interested in compilers.
+"""
++++
+
+## C++ Infrastructure for Bril
+Albert Xiao & Ryan Mao
```

Comment:
You don't need to include the title & authors; these will appear automatically when rendered.